ansible/roles/uan_haproxy/templates/haproxy-values.yml.j2

```diff
@@ -0,0 +1,34 @@
+kind: {{ haproxy_kind|default("DaemonSet") }}
+service:
+  type: {{ haproxy_service_type|default("LoadBalancer") }}
+containerPorts:
+  ssh: 22
+  http: null
+  https: null
+  stat: null
+config: |
+{% if haproxy_config|length>0 %}
+{{ haproxy_config|indent(2, true) }}
+{% else %}
+  global
+    log stdout format raw local0
+    maxconn 1024
+
+  defaults
+    log     global
+    mode    tcp
+    timeout connect 10s
+    timeout client 36h
+    timeout server 36h
+    option  dontlognull
+
+  listen ssh
+    bind *:22
+    balance leastconn
+    mode tcp
+
+    option tcp-check
+    tcp-check expect rstring SSH-2.0-OpenSSH.*
+
+    server uan01 10.103.11.142:9001 check inter 10s fall 2 rise 1
```

Comment:
Just wondering if the IP address here is always going to be this value or whether it should be a parameter?

Reply:
That shouldn't be there. Good catch. This is the default config block if nothing is defined. I do need a way to correctly populate this with values for a real uan though based on the ansible facts.